docs/3.api/1.composables/use-seo-meta.md

```diff
@@ -0,0 +1,44 @@
+---
+description: The useSeoMeta composable lets you define your sites SEO Meta as a flat object with full TypeScript support
+---
+
+# `useSeoMeta`
+
+The `useSeoMeta` composable lets you define your site's SEO meta tags as a flat object with full TypeScript support.
+
+This helps you avoid common mistakes, such as using `name` instead of `property` as well typos with over 100+ meta tags fully typed.
```

Comment:
```suggestion
This helps you avoid common mistakes, such as using `name` instead of `property`, as well as typos - with over 100+ meta tags fully typed.
```

Reply:
I've just copied the text from here [https://unhead.harlanzw.com/guide/composables/use-seo-meta](https://unhead.harlanzw.com/guide/composables/use-seo-meta), maybe I can create PR there also 🙂

Reply:
good idea!

also note that `docs/1.getting-started/5.seo-meta.md` has a lot of similar content....

Reply:
I agree, it's almost the sam, but I find it a bit confusing to find it because it's only mentioned under the **getting started** section.

A month ago, I've forgotten the name of the composable and automatically started searching for it under **composables** and it wasn't there. If you want, I can write a little bit more details about it.

Reply:
I think it makes sense - but probably would be a good idea to link between pages and de-duplicate a bit

Reply:
Sure, I'll try to figure something out